src/providers/seventv/SeventvEmotes.hpp

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include "SeventvEmoteCache.hpp"
```

Comment:
Should be
```suggestion
#include "providers/seventv/SeventvEmoteCache.hpp"
```
I'll do that next push.